### ISSUE TITLE
Face.geom_type() returns string rather than GeomType

### ISSUE THREAD

**Issue by MatthiasJ1** [gumyr/build123d#466]
`Face.geom_type()` might also make more sense as `@property`.

```py
print(f"{Rectangle(1,1).face().geom_type() = }")
print(f"{GeomType.PLANE = }")
print(f"{Rectangle(1,1).face().geom_type()==GeomType.PLANE = }")
```
```
Rectangle(1,1).face().geom_type() = 'PLANE'
GeomType.PLANE = <GeomType.PLANE>
Rectangle(1,1).face().geom_type()==GeomType.PLANE = False
```

**Comment by gumyr** [comment]
Duplicate of Issue #321

**Comment by jdegenstein** [comment]
Fixed with PR #559
```py
from build123d import *
print(f"{Rectangle(1,1).face().geom_type = }")
print(f"{GeomType.PLANE = }")
print(f"{Rectangle(1,1).face().geom_type==GeomType.PLANE = }")
```
now returns
```py
Rectangle(1,1).face().geom_type = <GeomType.PLANE>
GeomType.PLANE = <GeomType.PLANE>
Rectangle(1,1).face().geom_type==GeomType.PLANE = True
```
